deploy/cephfs/kubernetes/v1.13/helm/ceph-csi-cephfs/Chart.yaml

```diff
@@ -12,3 +12,4 @@ keywords:
 home: https://github.com/ceph/ceph-csi
 sources:
   - https://github.com/ceph/ceph-csi/tree/csi-v1.0/deploy/cephfs/helm
+icon: https://raw.githubusercontent.com/ceph/ceph-csi/master/assets/ceph-logo.png
```

Comment:
is this logo present? it's giving 404 error

Reply:
I think `master` is not required ?

Reply:
Based if of an existing file:
https://raw.githubusercontent.com/ceph/ceph-csi/master/.github/pull_request_template.md

@humblec Master is required otherwise it returns a 404:
https://raw.githubusercontent.com/ceph/ceph-csi/.github/pull_request_template.md

Reply:
I could split the adding of the logo to a separate PR so we are able to test the url before this is merged?

Reply:
can we use an existing logos from ceph.com or we have to maintain one ceph-csi?
sample link https://ceph.com/wp-content/uploads/2016/07/Ceph_Logo_Stacked_RGB_120411_fa.png

Reply:
We could link to that logo but it breaks when the ceph.com site changes (moves away from Wordpress for example). If it was ceph.com/logo/logo_stacked.png I would have gone for that but by adding the logo we ensure a valid link.